api-v2/src/plan/entities/plan.entity.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { CourseWarning, IWarning } from '../../../../frontend/src/models/types';
+import { Schedule } from '../../../../common/types';
+import { Student } from '../../student/entities/student.entity';
+
+@Entity()
+export class Plan {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @ManyToOne(
+    () => Student,
+    student => student.plans,
+    {
+      cascade: true,
+      onDelete: 'CASCADE',
+    },
+  )
+  student: Student;
+
+  @Column({ type: 'json' })
+  schedule: Schedule;
+
+  @Column()
+  major: string;
+
+  @Column()
+  coopCycle: string;
+
+  @Column({ nullable: true })
+  concentration: string;
+
+  @Column({ type: 'smallint', default: 2018 })
+  catalogYear: number;
+
+  @Column({ type: 'json' })
+  courseWarnings: CourseWarning[];
+
+  @Column({ type: 'json' })
+  warnings: IWarning[];
```

Comment:
i think it's fine to omit these, since we only calculate them on the frontend – does the current backend save them?

Reply:
Yep the current backend does save them. I kinda just copy pasta'd the entity shapes but there's no reason as to why we shouldn't revise them. I don't see any particular reason as to why we should keep them so I can strip them out. We can always bring them back if we need to? Any thoughts?

Reply:
Sounds good to remove 👍